src/content/overlay.ts

```diff
@@ -137,6 +137,16 @@ export default class Overlay {
     }
   }
 
+  private clickNode(node: Node) {
+    if ((node as HTMLElement).className.includes("CodeMirror")) {
```

Comment:
minor thing, but I think you can do
```
const element = node as HTMLElement;
```

and then you don't have to repeat the cast everywhere.